habitat_baselines/common/utils.py

```diff
@@ -53,6 +56,51 @@ def forward(self, x):
         return CustomFixedCategorical(logits=x)
 
 
+class ResizeCenterCropper(nn.Module):
+    def __init__(
+        self, force_input_size: Optional[tuple], channels_first: bool = True
+    ):
+        super().__init__()
+        assert (
+            len(force_input_size) == 2
+        ), "forced input size must be len of 2 (h, w)"
+        self.force_input_size = force_input_size
+        self.channels_first = channels_first
+
+    def transform_observation_space(
+        self, observation_space, trans_keys=["rgb", "depth", "semantic"]
+    ):
+        force_input_size = self.force_input_size
+        observation_space = copy.deepcopy(observation_space)
+        if force_input_size:
+            for key in observation_space.spaces:
+                if key in trans_keys:
+                    logger.info(
+                        "Overwriting CNN input size of %s: %s"
+                        % (key, force_input_size)
+                    )
+                    observation_space.spaces[key] = overwrite_gym_box(
+                        observation_space.spaces[key], force_input_size
+                    )
+        self.observation_space = observation_space
+        return observation_space
+
+    def forward(self, input: torch.Tensor):
+        if self.force_input_size is None:
+            return input
+
+        return center_crop(
+            image_resize_shortest_edge(
```

Comment:
How to disable `image_resize_shortest_edge` functionality in current setup?

Reply:
There isn't a way, that's why it's ResizeCenterCropper not just CenterCropper. Currently there is no way to disable it.